src/playdate/sound.nim

```diff
@@ -139,6 +140,11 @@ proc privateFilePlayerFinishCallback(soundSource: SoundSourcePtr, userdata: poin
     if filePlayer.finishCallback != nil:
         filePlayer.finishCallback(filePlayer)
 
+proc privateFilePlayerFadeVolumeCallback(soundSource: SoundSourcePtr, userdata: pointer) {.cdecl, raises: [].} =
```

Comment:
Should we be doing something with SoundSourcPtr? Passing that into callback, for example?